Makefile

```
@@ -0,0 +1,218 @@
+
+.PHONY: markdownlint
+markdownlint:
+	@echo -e "\n$(YELLOW)Running markdownlint...$(NC)\n"
+	@which markdownlint-cli2 >/dev/null 2>&1 || { \
+		echo "$(RED)Error: 'markdownlint-cli2' is not installed. Please install it first.$(NC)"; \
+		exit 1; \
+	}
+	@markdownlint-cli2 .
+
+.PHONY: chktex
+chktex:
```

checkmake (0.2.2)

[warning] Target body for "chktex" exceeds allowed length of 5 (6).

(maxbodylength)